scripts/huggingface.py

```diff
@@ -149,7 +149,7 @@ def __init__(
         self.model_format = model_format
         if self.model_format == "neural_speed":
             self.use_quant=True
-            if weight_dtype=="fp32":
+            if weight_dtype=="fp32" and use_autoround==False and use_gptq==False and use_awq==False:
```

Comment:
Thes `use_xxx` args will be removed later. So please avoid relying on them too much.

Reply:
OK